SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.iOS/ErrorReportCache.cs

```diff
@@ -1,23 +0,0 @@
-﻿using System.Collections.Generic;
```

Comment:
Because the disk instances of wrapper exception are cached in memory by native SDK, so there's no need to complicate logic by unnecessarily caching